script.js

```diff
@@ -0,0 +1,115 @@
+let player = 1;
+const display = () => {
+    let endgame = false;
+
+    let currentClick = document.querySelectorAll('.grid-item')
+    for (let i = 0; i< currentClick.length && !endgame; i ++){
+         currentClick[i].addEventListener( 'click', (event) =>{
+        if (endgame) return;
+
+        if(currentClick[i].innerText === "X" || currentClick[i].innerText === "O"){
+          return;
+            }
+
+        if(player === 1){
+         
+         currentClick[i].innerText = ("X");
+         document.getElementById('print').innerText = " Player O turn ";
+         player = 0;
+        }
+        else{
+         currentClick[i].innerText = ("O")
+         document.getElementById('print').innerText = " Player X turn ";
+         player = 1;
+        }
+
+        const box1 = document.getElementById('box1') && document.getElementById('box1').innerText;
+        const box2 = document.getElementById('box2') && document.getElementById('box2').innerText;
+        const box3 = document.getElementById('box3') && document.getElementById('box3').innerText;
+        const box4 = document.getElementById('box4') && document.getElementById('box4').innerText;
+        const box5 = document.getElementById('box5') && document.getElementById('box5').innerText;
+        const box6 = document.getElementById('box6') && document.getElementById('box6').innerText;
+        const box7 = document.getElementById('box7') && document.getElementById('box7').innerText;
+        const box8 = document.getElementById('box8') && document.getElementById('box8').innerText;
+        const box9 = document.getElementById('box9') && document.getElementById('box9').innerText;
+   
+        if (box1 == "X" && box2 == "X" && box3 == "X"){
```

Comment:
I'm glad you got it to work the long way! That is always our first priority when coding. I would challenge you to try and reduce the repetition of these checks as a next level of understanding. One way would be to find a way to put the win conditions in a loop, this is pretty tough. An easier way to reduce repetition would be to put the print code and the endgame = true into a function, and call that function.